driver/src/lib.rs

```diff
@@ -11,7 +11,8 @@ use std::{
 
 pub mod error;
 
-const KNOWN_DEVICE_INFOS: [(u16, u16, u16, u16); 8] = [
+const KNOWN_DEVICE_INFOS: [(u16, u16, u16, u16); 9] = [
+    (0x048d, 0xc995, 0xff89, 0x00cc), // 2024 // Legion pro 5i - maybe all 2024?
```

Comment:
```suggestion
const KNOWN_DEVICE_INFOS: [(u16, u16, u16, u16); 10] = [
    (0x048d, 0xc995, 0xff89, 0x00cc), // 2024
    (0x048d, 0xc993, 0xff89, 0x00cc), // 2024 LOQ
```

It's probably safe to assume it'll "just work" on every Legion 5 this time around given how things have been going. Also going by this reasoning I added an entry for the LOQ/Ideapad variants.